app/src/shared/utils/withErrorBoundary.jsx

```diff
@@ -19,11 +19,14 @@ const withErrorBoundary = (ErrorComponent: ComponentType<any>) => (
                 error: undefined,
             }
 
-            static getDerivedStateFromError(error: Error, extra: any) {
-                console.error({
-                    error,
-                })
-                analytics.reportError(error, extra)
+            static getDerivedStateFromError(error: any, extra: any) {
+                if (error && !error.noReport) {
```

Comment:
If a thrown thing has a truthy `noReport` property then we skip logging and reporting it. Used maily for "expected" exceptions like `ResourceNotFoundError`.